code/game/machinery/camera/camera_assembly.dm

```diff
@@ -4,32 +4,34 @@
 	icon = 'icons/obj/monitors.dmi'
 	icon_state = "cameracase"
 	materials = list(MAT_METAL=400, MAT_GLASS=250)
-	result_path = /obj/machinery/camera_assembly
+	result_path = /obj/structure/camera_assembly
 
 
-/obj/machinery/camera_assembly
+/obj/structure/camera_assembly

```

Comment:
changed to a structure since it doesn't process anything, isn't powered. Same thing happened for light_construct.